kong.yml

```diff
@@ -90,7 +91,7 @@ nginx: |
     real_ip_recursive on;
 
     # Other Settings
-    client_max_body_size 128m;
+    client_max_body_size 0m;
```

Comment:
There should probably still be a max body size as it is more a matter of security than anything else. And we can document "Your request size limiting settings cannot exceed the maximum size configured in your kong configuration".